packages/main/src/Table.js

```diff
@@ -712,13 +757,117 @@ class Table extends UI5Element {
 		}
 	}
 
+	/**
+	 * Toggles focus between the table row's root and the last focused nested element.
+	 * @private
+	 * @param {CustomEvent} event "ui5-f7-pressed"
+	 */
+	_handleF7(event) {
+		const row = event.detail.row;
+		const activeElement = getActiveElement();
+		row._tabbables = getTabbableElements(row);
+
+		if (!row._tabbables.length) {
+			return;
+		}
+
+		if (activeElement === row.root) {
+			const lastFocusedElement = row._tabbables[this._prevNestedElementIndex];
+
+			if (lastFocusedElement) {
+				lastFocusedElement.focus();
+			} else {
+				row._tabbables[0].focus();
+			}
+
+			return;
+		}
+
+		const targetIndex = row._tabbables.indexOf(activeElement);
+
+		if (targetIndex > -1) {
+			this._prevNestedElementIndex = targetIndex;
+		}
+
+		row.focus();
+	}
```

Comment:
Here you can use if / else statement in general. 
However I think it can be further refactored to use less nested if statements...

Reply:
Done